bin/update-wdtk-rejections

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# update-wdtk-rejections:
+#
+#  Run WDTK Rake tasks to generate requests to block at MTA layer and
+#  add updated file to Puppet.
+#
+
+WDTK_DIR=${1:-/data/vhost/www.whatdotheyknow.com/alaveteli}
+
+# Are we running as root?
+if [ "$UID" != "0" ]; then
+  echo "This script should be run as root, aborting." >&2
+  exit 1
+fi
+
+# Is WDTK available where we think it is?
+if [ ! -d "$WDTK_DIR" ]; then
+  echo "WDTK doesn't appear to be availle in ${WDTK_DIR}, aborting." >&2
+  exit 2
+fi
+
+check_rake () {
+  RAKE_TASK=$1
+  RAKE_EXIT_CODE=$2
+  if [ "${RAKE_EXIT_CODE}" != "0" ]; then
```

Comment:
Should this (and others) use `-ne` and no quotes, as it's integer not string comparison?

Reply:
Yes!